modules/claims_api/app/controllers/concerns/claims_api/v2/disability_compensation_validation.rb

```diff
@@ -712,7 +711,7 @@ def validate_reserves_tos_dates!(reserves)
         end
       end
 
-      def validate_title_ten_activiation_values!(service_information)
+      def validate_title_ten_activation_values!(service_information)
```

Comment:
@stiehlrod Do you think it would be worth it here to change the method names from `_title_ten_` to `_federal_` ?  As a test I just did a mass reassign of `title_ten` to `federal` and everything looked good still. I realize it is a little nit-picky but seems like a related change to keep stuff inline, ? Everything else looked as expected, let me know your thoughts on it and I can approve.

Reply:
Changed